proto/node/akash/verification/v1/types.proto

```
@@ -0,0 +1,335 @@
+syntax = "proto3";
+package akash.verification.v1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.69.0)

[error] imported file does not exist

(COMPILE)